portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Oyin's Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+
+<style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+        }
+</style>
+
+
+<body>
+    <h1>"How Grace Hopper Impacted Me"</h1>
+    <h2>By: Oyin Sholeye</h2>
+    <h3>Posted on May 28, 2020</h3>
+    <img src="/images/IMG-3018.JPG" alt="GHC" style="width: 250px;height:300px" >
```

Comment:
You can upload the image file in this pull request too so its not missing!

Reply:
![IMG-3018](https://user-images.githubusercontent.com/42747431/83227677-61484700-a14a-11ea-8384-a4e7f4c3050a.JPG)

I am hoping this shows up.

Reply:
I think Jessica meant adding the image as a file in your branch :) 

This pull request currently only has your HTML file in it, you'll also want your image file to be a file in the request